Week2/js-exercises/ex4-collectiveAge.js

```diff
@@ -10,6 +10,9 @@
 
 function collectiveAge(people) {
   // return the sum of age for all the people
+  const ages = people.map(item => item.age)
+  const sumAges = ages.reduce((a,b) => a + b ,0)
```

Comment:
You can only use a reduce and no map. `people.reduce((total, person)=> total + person.age, 0)`

Reply:
In the exercise it said use map to get the ages.